brage-import/src/main/java/no/sikt/nva/brage/migration/record/Identity.java

```diff
@@ -10,11 +10,17 @@ public class Identity {
 
     private final String name;
     private final String identifier;
+    private final String orcId;
 
-    public Identity(@JsonProperty("name") String name,
-                    @JsonProperty("identifier") String identifier) {
+    public Identity(@JsonProperty("name") String name, @JsonProperty("identifier") String identifier,
```

Comment:
Beginning to agree with @LarsV123 at the code style should be stricter since we aren't fastidious in readable formatting. Break the parameters to new lines.

Reply:
Should we modify our codestyle scheme?